arm-customer-insights/2016-01-01/examples/InteractionsCreateOrUpdate.json

```diff
@@ -0,0 +1,181 @@
+{
+	"parameters": {
+		"hubName": "sdkTestHub",
+		"resourceGroupName": "TestHubRG",
+		"interactionName": "TestProfileType396",
+		"api-version": "2016-01-01",
+		"subscriptionId": "subid",
+		"parameters": {
+			"location": "West US",
+			"properties": {
+				"idPropertyNames": ["TestInteractionType6358"],
+				"primaryParticipantProfilePropertyName": "profile1",
+				"apiEntitySetName": "TestInteractionType6358",
+				"entityType": "Interaction",
+				"fields": [{
+					"fieldName": "TestInteractionType6358",
+					"fieldType": "Edm.String",
+					"isArray": false,
+					"isRequired": true
+				},
+				{
+					"fieldName": "profile1",
+					"fieldType": "Edm.String"
+				}],
+				"smallImage": "\\\\Images\\\\smallImage",
+				"mediumImage": "\\\\Images\\\\MediumImage",
+				"largeImage": "\\\\Images\\\\LargeImage"
+			}
+		}
+	},
+	"responses": {
+		"200": {
+			"body": {
+				{
```

Comment:
unnecessary curly braces. The "body" key points to an object, no need to wrap it again inside curly braces. please remove the opening and closing curly braces.